### ISSUE TITLE
Cannot load PKCS11 module on macOS ("file not found")

### ISSUE THREAD

**Issue by KizzyCode** [chris2511/xca#347]
XCA fails to open PKCS#11 libraries (["file not found"](https://user-images.githubusercontent.com/13999009/161408045-69249a7c-7124-4800-aa8f-99868cf5801d.png)), even if the file exists (i.e. `softhsm2.so` and `libykcs11.dylib`).
This issue may be related to #292, however none of the workarounds worked so it might not be connected at all.

---

**Things I've tried:**
 - Remove the signature and re-sign with my own certificate and [the entitlement from #292](https://github.com/chris2511/xca/issues/292#issuecomment-884362917).
 - Use XCA 2.3.0.78 instead

Furthermore, there are no `Library Validation failed`-errors in my console.

---

**System info:**
 - MacBook Pro (14-inch, 2021), Apple M1 Pro
 - macOS Monterey 12.3.1 (21E258)

**Comment by foxt** [comment]
Work around works for me:

```
codesign --remove-signature /Applications/xca.app/
codesign --sign - /Applications/xca.app/
```

Of course, this will weaken the security slightly by removing the signature.

**Comment by chris2511** [comment]
Closed with #325
